.pylintrc

```diff
@@ -13,6 +13,11 @@ load-plugins=pylint.extensions.bad_builtin,
              pylint.extensions.redefined_variable_type,
 
 [MESSAGES CONTROL]
+enable=
+    bad-inline-option,
+    deprecated-pragma,
+    useless-suppression,
+    use-symbolic-message-instead,
```

Comment:
If you want to know what a PyLint message does by the way:

```terminal
$ tox -qqe lint --run-command 'pylint --help-msg use-symbolic-message-instead'
```

And for a list of all the messages:

```terminal
$ tox -qqe lint --run-command 'pylint --list-msgs'
```

(Trying to find PyLint docs online is kind of hopeless.)

Reply:
I just came to this conclusion. They only seem to mention them in change logs